dulwich/patch.py

```diff
@@ -207,11 +207,17 @@ def lines(content):
     new_content = content(new_mode, new_id)
     if not diff_binary and (
             is_binary(old_content.data) or is_binary(new_content.data)):
-        f.write(b"Binary files " + old_path + b" and " + new_path +
-                b" differ\n")
+        binary_diff = (
```

Comment:
Please keep changes minimal, there shouldn't be a need to reflow these lines in a style that's different from the surrounding code.